one-member.php

```diff
@@ -0,0 +1,99 @@
+<?php require 'connection.php'; ?>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>One Member</title>
+    <link rel="stylesheet" href="bootstrap.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <?php
+            $memberID = $_GET["memberID"];
```

Comment:
You must check if the parameter exists before assigning to the variable.

Reply:
OK